presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -197,20 +197,37 @@ public TaskStatus process()
 
```

Comment:
`When writing a Git commit message, follow these guidelines` https://chris.beams.io/posts/git-commit/, please see: https://github.com/prestodb/presto#developers

Reply:
Updated the commit message.